src/N98/Magento/Command/Cache/ReportCommand.php

```diff
@@ -0,0 +1,162 @@
+<?php
+
+namespace N98\Magento\Command\Cache;
+
+use Magento\Framework\App\CacheInterface;
+use Magento\PageCache\Model\Cache\Type as FullPageCache;
+use N98\Magento\Command\AbstractMagentoCommand;
+use N98\Util\Console\Helper\Table\Renderer\RendererFactory;
+use Symfony\Component\Console\Input\InputInterface;
+use Symfony\Component\Console\Input\InputOption;
+use Symfony\Component\Console\Output\OutputInterface;
+use Symfony\Component\Validator\Exception\RuntimeException;
+
+class ReportCommand extends AbstractMagentoCommand
+{
+    /**
+     * @var CacheInterface
+     */
+    private $cache;
+
+    /**
+     * @var FullPageCache
+     */
+    private $fpc;
+
+    /**
+     * @param array $metadata
+     * @param InputInterface $input
+     * @return bool
+     */
+    private function isTagFiltered($metadata, $input)
```

Comment:
I know you prefer to define before use, but if you could move the private method below the public (or here: protected execute()) method that calls it would be great. :)